### ISSUE TITLE
Inline `in_batches` false-positive on `Migration/BatchInBatches`

### ISSUE THREAD

**Issue by barak-neeman** [r7kamura/rubocop-migration#14]
```Model.where(field: nil).in_batches.update_all(field: false)```

Gets flagged by the Migration/BatchInBatches cop, and auto-corrected to:

```
Model.where(field: nil).in_batches.in_batches do |relation|
  relation.update_all(field: false)
end
```

Please update the Migration/BatchInBatches cop to allow for the above inline usage of in_batches, as the correction is unnecessary and produces code with two consecutive in_batches calls.

**Comment by r7kamura** [comment]
Thank you for your report! I completely forgot about that when I implemented it.

Maybe if we improve in these lines, I think it will work also in inline calls:

https://github.com/r7kamura/rubocop-migration/blob/14c09ee6e5fad793bc06d552d074c97245552994/lib/rubocop/cop/migration/batch_in_batches.rb#L88-L91

**Comment by r7kamura** [comment]
Maybe we should also use the inline style for autocorrection?

```ruby
# bad
User.update_all(some_column: 'some value')

# good
User.in_batches.update_all(some_column: 'some value')
```

**Comment by r7kamura** [comment]
Added support for inline `in_batches` at:

- https://github.com/r7kamura/rubocop-migration/pull/16

and also changed its autocorrection result to prefer the inline style over the block style at:

- https://github.com/r7kamura/rubocop-migration/pull/17

**Comment by r7kamura** [comment]
Just published v0.5.0:

- https://github.com/r7kamura/rubocop-migration/releases/tag/v0.5.0
- https://rubygems.org/gems/rubocop-migration/versions/0.5.0

It was a good opportunity to improve `Migration/BatchInBatches`. Thanks again!

**Comment by barak-neeman** [comment]
Update works for all my test cases, and amazingly fast too!
Thank you very much!!
